monix-reactive/shared/src/test/scala/monix/reactive/internal/operators/ThrottleLatestSuite.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2014-2021 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.internal.operators
+
+import monix.execution.Ack.Continue
+import monix.reactive.{Observable, Observer}
+import monix.reactive.internal.operators.MaxSuite.{assert, assertEquals, test}
+
+import scala.concurrent.duration._
+
+object ThrottleLatestSuite extends BaseOperatorSuite {
+  def sum(sourceCount: Int) = {
+    sourceCount * (sourceCount - 1) / 2
+  }
+  def createObservable(sourceCount: Int) = Some {
+    if (sourceCount == 1) {
+      val o = Observable.now(100L).delayExecution(500.millis).throttleLatest(1.second, true)
+      Sample(o, 1, 100, 500.millis, 1.second)
```

Comment:
Please test with something with more than one element in the output :D That might prove that emitLast handling is correct but does very little to actually prove that throttling is working as expected.

Reply:
I don't think we are tsting with one element on the output here :) I think `createObservable` gets called from outside and we pass the `sourceCount` from`BaseOperatorSuite`, so that is randomized. 
I've added this `sourceCount ==1` check because I've noticed that in several suites, but now that I've started chcecking I think it's not needed :)

Reply:
Right. Disregard that line comment but please adjust your additional tests to have more than 1-2 elements :)

Reply:
Sure, sounds reasonable I will add few more tests.

Reply:
I think we could still test cases from `SampleOnceSuite`

Reply:
Sure, gonna take a look at them.